packages/nameguard-react/src/styles.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Blank line needed. Seems to be the only update needed in this PR. Great work with the prefix config 👍🏼